File: python/rosettacode_org/Associative_array/Merging/merging.js
(() => {
    'use strict';

    console.log(JSON.stringify(
        Object.assign({},   // Fresh dictinoary
        { // Base
            "name": "Rocket Skates",
            "price": 12.75,
            "color": "yello"
        }, { //update.
            "price": 15.25,
            "color": "red",
            "year": 1974
        }
        ),
        null, 2
    ))
})();

File: rosettacode_org/A/Associative_array/Merging/merging.py
base = {"name":"Rocket Skates", "price":12.75, "color":"yellow"}
update = {"price":15.25, "color":"red", "year":1974}

result = {**base, **update}

print(result)


base = {"name":"Rocket Skates", "price":12.75, "color":"yellow"}
update = {"price":15.25, "color":"red", "year":1974}

result = base.copy()
result.update(update)

print(result)

base = {"name":"Rocket Skates", "price":12.75, "color":"yellow"}
update = {"price":15.25, "color":"red", "year":1974}
result = base | update
print(result)

